assets/js/components/KeyMetrics/MetricsSelectionPanel/Header.js

```diff
@@ -57,7 +60,15 @@ export default function Header( { closePanel } ) {
 
 	return (
 		<SelectionPanelHeader
-			title={ __( 'Select your metrics', 'google-site-kit' ) }
+			title={
+				isConversionReportingEnabled
+					? sprintf(
+							/* translators: %d: number of max allowed metrics. */
+							__( 'Select up to %d metrics', 'google-site-kit' ),
+							MAX_SELECTED_METRICS_COUNT_WITH_CONVERSION_EVENTS
```

Comment:
This should use `_n` given it's dealing with numbers; if `MAX_SELECTED_METRICS_COUNT_WITH_CONVERSION_EVENTS` was `1` this should be `Select up to %d metric` instead. And other locales can have complex plural rules, so let's update this to use `_n`